pkgs/by-name/gr/grayjay/package.nix

```diff
@@ -35,15 +35,14 @@
   _experimental-update-script-combinators,
 }:
 let
-  version = "10";
+  version = "11";
   src = fetchFromGitLab {
     domain = "gitlab.futo.org";
     owner = "videostreaming";
     repo = "Grayjay.Desktop";
     tag = version;
-    hash = "sha256-ap0NnjyBjvyFjHPu9vACQMoOXqwz90/8QqSfPFqfh5U=";
+    hash = "sha256-UIH67Ky2IJVF45kK6tHFT5nPMfytDsgAfece0Tv8cIE=";
     fetchSubmodules = true;
-    fetchLFS = true;
```

Comment:
LFS is required because it brings in `dotcefnative`. Unfortunately, I can't leave a suggestion but I was able to build and run using this:
```
hash = "sha256-YJlUxoeGodPwPabLN+g0YOJ7dTZgFxJqyQGu5SYk0J8=";
fetchSubmodules = true;
fetchLFS = true;
```